gkeys/gkeys/email_reminders.py

```diff
@@ -47,14 +49,15 @@ def _send_email(self, message):
         mail = smtplib.SMTP('smtp.gmail.com', 587)
         mail.ehlo()
         mail.starttls()
-        mail.login('aperivol@gmail.com', 'PASSWORD')
+        email_from = self.config.get_key('login_email')
+        mail.login(email_from, self.config.get_key('login_password'))
```

Comment:
this code only allows for one login, I specified in my earlier comments to add a--from option to specify the send user info in the email.conf.  Each user has it's own info login id, password, etc...  That way the config file can have as many user's to be able to send as as there are config values for.

# user email accounts.
[foo]
login = foo
passwd = 123456asdf
server = smtp.gentoo.org

[bar]
login = bar
password = 123zxcv
server = smtp.gmail.com

...